### ISSUE TITLE
`</pre><pre>` converts to 6 backticks

### ISSUE THREAD

**Issue by captn3m0** [thephpleague/html-to-markdown#145]
```php
$c = new HtmlConverter();
echo $c->convert("<pre>\n</pre><pre>\n</pre>");
```

Renders 6 backticks.

**Comment by colinodell** [comment]
It looks like it failed to place a newline after the first `</pre>` was parsed:

    ```
    
    ``````
    
    ```

This incorrect Markdown converts back to this HTML:

    <pre><code>
    </code></pre>
    <pre><code></code></pre>

When it should have been:


    ```
    
    ```
    ```
    
    ```

Which essentially converts back to what you had:

    <pre><code>
    </code></pre>
    <pre><code>
    </code></pre>

(Note that CommonMark also wraps the newline with `<code>...</code>`)

**Comment by colinodell** [comment]
Just released [v4.6.1](https://github.com/thephpleague/html-to-markdown/releases/tag/4.6.1) with a fix for this :)

**Comment by captn3m0** [comment]
Thanks a lot for the quick fix!
